script.module.resolveurl/lib/resolveurl/plugins/mixdrop.py

```diff
@@ -24,9 +24,9 @@
 
 class MixDropResolver(ResolveUrl):
     name = 'MixDrop'
-    domains = ['mixdrop.co', 'mixdrop.to', 'mixdrop.sx', 'mixdrop.bz', 'mixdrop.ch',
+    domains = ['mixdrop.co', 'mixdrop.to', 'mixdrop.sx', 'mixdrop.bz', 'mixdrop.ch','mixdrop.club',
```

Comment:
Please follow Flake8 linting. need space after comma
Also there are already 5 items in that line, please add it to the next line after `mixdrop.gl`

Reply:
ok,no problem